spec/requests/pages_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe PagesController, type: :request do
+  let(:policy) { GetIntoTeachingApiClient::PrivacyPolicy.new(id: "123", text: "Latest privacy policy") }
+  let(:policy_id) { 123 }
```

Comment:
If policy_id is a UUID v4 in CRM, this could be swapped out with `SecureRandom.uuid` ?

Reply:
Done